<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>594</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>922</width>
    <height>594</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>922</width>
    <height>594</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>550</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/new/prefix1/image/img1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="prePushBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>←上一张</string>
   </property>
  </widget>
  <widget class="QPushButton" name="randomPushBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>520</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>试手气</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPushBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>260</y>
     <width>91</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>下一张→</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imgName">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>电子相册</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_imgName_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>第</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imgName_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>480</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>张  共5张</string>
   </property>
  </widget>
  <widget class="QPushButton" name="carousePushBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>停止轮播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>520</y>
     <width>112</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭窗口</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdDate">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>19</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
